src/components/InfoModal.js

```diff
@@ -46,13 +46,14 @@ class InfoModal extends PureComponent<Props> {
       children,
       confirmLabel,
       confirmProps,
+      style,
     } = this.props;
     return (
       <BottomModal
         id={id}
         isOpened={isOpened}
         onClose={onClose}
-        style={styles.modal}
+        style={[styles.modal, style || {}]}
```

Comment:
(minor: RN is totally fine with a falsy value in style)

Reply:
TIL 😯